src/server/services/varen/decos-zaken.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  DecosZaakTransformer,
+  SELECT_FIELDS_TRANSFORM_BASE,
+} from '../decos/decos-types';
+import type {
+  CaseTypeVaren,
+  VarenRegistratieRederType,
+  VarenVergunningExploitatieType,
+  VarenVergunningLigplaatsType,
+} from './config-and-types';
+
+export const VarenRegistratieReder: DecosZaakTransformer<VarenRegistratieRederType> =
+  {
+    isActive: true,
+    caseType: 'Varen registratie reder',
+    title: 'Varen registratie reder',
+    fetchWorkflowStatusDatesFor: [
+      // TODO: Stapnaam komt nog door vanuit bronsysteem
+      {
+        status: 'In behandeling',
+        stepTitle: 'Varen - Behandelen',
+      },
+    ],
+    transformFields: {
+      ...SELECT_FIELDS_TRANSFORM_BASE,
+      company: 'company',
+      num2: 'bsnkvk',
+      mailaddress: 'adres',
```

Comment:
Ik denk dat we hier beter engels kunnen gebruiken. Neem de vuistregel: engels tenzij ... een term zich beter leent in het Nederlands / Specifiek is gekoppeld aan een domein ... `address`